ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -193,3 +193,59 @@ extension TabBarViewDemoController: TabBarViewDelegate {
         present(alert, animated: true)
     }
 }
+
+// MARK: - TabBarViewDemoController: DemoAppearanceDelegate
+extension TabBarViewDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        var tokensClosure: ((TabBarView) -> TabBarTokens)?
+        if isOverrideEnabled {
+            tokensClosure = { _ in
+                return ThemeWideOverrideTabBarTokens()
+            }
+        }
+
+        fluentTheme.register(controlType: TabBarView.self, tokens: tokensClosure)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        let tokens = (isOverrideEnabled ? PerControlOverrideTabBarItemTokens() : nil)
+        _ = tabBarView?.overrideTokens(tokens)
```

Comment:
Probably outside the scope of this PR, but it's strange to me that the tabBarView of the TabBarViewDemo is optional?

Reply:
Definitely outside the scope of this PR, though there are other demo issues I'd like to clear up (like the leading badge not disappearing after toggling badges off), so I will try to tackle this issue in another change.